### ISSUE TITLE
Update to React Native 0.60 (Windows)

### ISSUE THREAD

**Issue by nickwinder** [PSPDFKit/react-native#294]
# Details
When possible we need to update windows to react-native 0.60 to bring the platform in line with iOS and Android. 

Currently we are blocked from doing this work because `react-native-windows` does not support 0.60 yet. It is planned for Summer-fall, but no word yet. See question https://github.com/microsoft/react-native-windows/issues/2815#issuecomment-527467218. 

When this work is complete we can drop the `windows-support` branch and pull windows back into `master`.

# Acceptance Criteria

- [ ] Upgrade windows to support 0.60
- [ ] Pull `windows-support` branch back into `master`

**Comment by steipete** [comment]
Looking at https://github.com/microsoft/react-native-windows/issues/2786 they target end of October for the release.

**Comment by nickwinder** [comment]
@steipete This is specifically for `vnext` not for the current `master` release. It seems we have no hard date for that. @leoek asked the question directly.

**Comment by harinikmsft** [comment]
Do you need the upgrade for `current`? We just completed the upgrade to 0.60 for `vnext`. See [PR](https://github.com/microsoft/react-native-windows/issues/2786)

**Comment by leoek** [comment]
@harinikmsft Thanks for the feedback. Unfortunately we do need it for `current` as I believe that @nickwinder still needs time until a migration to `vnext` is ready.

**Comment by nickwinder** [comment]
@leoek @harinikmsft Yes we would not consider migrating to `vnext` until we can be confident with its stability. I would assume this would be around the time a `vnext` RC is ready.

**Comment by harinikmsft** [comment]
Thanks, that makes sense. Let me circle back with the crew on our side and get back to you on timelines for upgrading `current` to RN 0.60.

**Comment by leoek** [comment]
@harinikmsft any news yet?

**Comment by nickwinder** [comment]
Looks like we're still looking at 30th June 2020 for full Core API support in RNW. https://github.com/microsoft/react-native-windows/milestone/19

**Comment by leoek** [comment]
Hi @nickwinder @steipete https://github.com/microsoft/react-native-windows/milestone/18 is now completed an vnext is the new default version for react-native-windows. Also according to microsoft they've reached feature parity with vcurrent and libraries should start to migrate to vnext :tada: 

What is the eta for the migration of react-native-pspdfkit to vnext?

Please note that we'll need an update for reac-tnative-pspdfkit asap. There are a lot of bugs (android and ios version too) on the last version with windows support.

**Comment by harinikmsft** [comment]
@leoek thanks for the ping. Yes, indeed. With the upgrade to v0.61, vnext is considered complete enough for most apps and scenarios and we released our first stable version, you can read more about it in our [blog](https://microsoft.github.io/react-native-windows/blog/2020/02/010/m4updates). We will be continuing to build out more support with future milestones.

**Comment by leoek** [comment]
@nickwinder @steipete any feedback?

**Comment by nickwinder** [comment]
@leoek We're looking into it, and we'll get back to you shortly. Thanks

**Comment by pweiskircher** [comment]
@leoek Unfortunately, we don’t have this on our roadmap for Q2. Based on the features we currently have in place to be built along with customers request, we can’t allocate the additional resources for this at this time.

We are more than happy to review and guide any implementation undertaken from your side as our React Native Wrapper is entirely open-source to allow clients of ours to contribute to its development! While I’m sure this isn’t your preferred outcome, it is unfortunately the only thing we can offer at this time.

**Comment by leoek** [comment]
Hi @pweiskircher,

this is really unfortunate. Support for react-native-windows was one of our main reasons for choosing pspdfkit.

> We are more than happy to review and guide any implementation undertaken from your side as our React Native Wrapper is entirely open-source to allow clients of ours to contribute to its development!

I think you misunderstood the spirit of open source development slightly. The code of the react-native-wrapper is indeed publicly accessible, however it is still copyrighted by Pspdfkit GmbH... Contributing to your react-native-wrapper with it's current license is pretty much free (unpaid) work for Pspdfkit GmbH.

Another issue we would encounter pretty fast, if we would start the implementation on our own is that only the wrapper itself is public and we would need access to the windows sdk to successfully port the wrapper to vnext.
